pkgs/by-name/fr/frequest/package.nix

```diff
@@ -0,0 +1,66 @@
+{ lib, stdenv, fetchFromGitHub, qt5 }:
+
+let
+  version = "1.2a";
+
+  src = fetchFromGitHub {
+    owner = "fabiobento512";
+    name = "FRequest";
+    repo = "FRequest";
+    rev = "v${version}";
+    hash = "sha256-fdn3MK5GWBOhJjpMtRaytO9EsVzz6KJknDhqWtAyXCc=";
+  };
+
+  commonLibsSrc = fetchFromGitHub {
+    owner = "fabiobento512";
+    name = "CommonLibs";
+    repo = "CommonLibs";
+    rev = "d3906931bb06ddf4194ff711a59e1dcff80fa82f";
+    hash = "sha256-iLJJ95yJ+VjNPuk8fNEDvYBI0db0rcfJF12a9azGv1Y=";
+  };
+
+  commonUtilsSrc = fetchFromGitHub {
+    owner = "fabiobento512";
+    name = "CommonUtils";
+    repo = "CommonUtils";
+    rev = "53970984f6538d78350be1b9426032bcb5bcf818";
+    hash = "sha256-nRv9DriSOuAiWhy+KkOVNEz5oSgNNNJZqk8sNwgbx8U=";
+  };
+
+in stdenv.mkDerivation {
+
+  pname = "frequest";
+  version = version;
+
+  srcs = [
+    src
+    commonLibsSrc
+    commonUtilsSrc
+  ];
+  sourceRoot = src.name;
```

Comment:
Isn't this the default?

Reply:
@SuperSandro2000 , removing it fails because the unpacker doesn't know which one to choose:

```
unpacking source archive /nix/store/f21vwbwsnxdj71wpx475pzh1xgg4yv4a-FRequest
unpacking source archive /nix/store/7g01fhxrb2cn9vr8wmr09f7mjjsxiy5y-CommonLibs
unpacking source archive /nix/store/h47hr43mchwz7jx6579gyk27b9a605z1-CommonUtils
unpacker produced multiple directories
```

If I remove one of all the names, same happens. If I remove all the names:

```
unpacking source archive /nix/store/2gc9m307sv0hmx40qxnrblzpr50kccqz-source
unpacking source archive /nix/store/vdy7vjh8z93vn8s4zqg4klm2dncb5s3w-source
Cannot copy /nix/store/vdy7vjh8z93vn8s4zqg4klm2dncb5s3w-source to source: destination already exists!
Did you specify two "srcs" with the same "name"?
```

Reply:
ah, I see, when specifying srcs this is required